src/sparseml/transformers/utils/helpers.py

```diff
@@ -58,14 +58,16 @@ class TaskNames(Enum):
     text_generation = {"text-generation"}
 
 
+ONNX_MODEL_NAME_INTERMEDIATE = "model-orig.onnx"
 RECIPE_NAME = "recipe.yaml"
 MANDATORY_DEPLOYMENT_FILES = {
     ONNX_MODEL_NAME,
     "tokenizer_config.json",
     "config.json",
 }
-NLG_TOKENIZER_FILES = {"special_tokens_map.json", "vocab.json", "merges.txt"}
 OPTIONAL_DEPLOYMENT_FILES = {"tokenizer.json", "tokenizer.model"}
+NLG_MANDATORY_DEPLOYMENT_FILES = {"special_tokens_map.json", "vocab.json", "merges.txt"}
```

Comment:
with we assert somewhere if these files aren't available from the original model? i'm pretty sure i've seen several models without these files, such as llama chat https://huggingface.co/meta-llama/Llama-2-7b-chat-hf/tree/main there is no vocab or merges file

Reply:
@mgoin this is outside of the scope of this PR. I moved some of those files from mandatory to optional in #1962